backend/frontend/templates/frontend/components/main_header.html

```diff
@@ -1,43 +1,69 @@
-
-    <div class="bg-gray-800">
-      <nav class="bg-gray-800">
-        <div class="mx-auto max-w-7xl sm:px-6 lg:px-8">
-          <div class="border-b border-gray-700">
-            <div class="flex h-16 items-center justify-between px-4 sm:px-0">
-              <div class="flex items-center">
-                <div class="flex-shrink-0">
-                  <img class="h-8 w-8" src="https://tailwindui.com/img/logos/mark.svg?color=indigo&shade=500" alt="Your Company">
-                </div>
-                <div class="hidden md:block">
-                  <div class="ml-10 flex items-baseline space-x-4">
-                    <!-- Current: "bg-gray-900 text-white", Default: "text-gray-300 hover:bg-gray-700 hover:text-white" -->
-                    <a href="#" class="bg-gray-900 text-white rounded-md px-3 py-2 text-sm font-medium" aria-current="page">Dashboard</a>
-                    <a href="#" class="text-gray-300 hover:bg-gray-700 hover:text-white rounded-md px-3 py-2 text-sm font-medium">Team</a>
-                    
-                  </div>
-                </div>
+<div class="bg-gray-800">
```

Comment:
Did you just change the indentation on this file? Or is there something to review here?

Reply:
I put sign_out actions in there, prettier must've made the diff huge on save